samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -36,8 +36,8 @@
     //  - Set Key using dotnet's user secrets (see above) (i.e. dotnet user-secrets set "Completion:Key" "MY_OPENAI")
```

Comment:
The change looks good to me,

But, for my own situational awareness, why are we doing ity?